geomesa-jobs/src/main/java/org/locationtech/geomesa/jobs/interop/mapred/FeatureCountJob.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2015 Commonwealth Computer Research, Inc.
+ *
+ * Licensed under the Apache License, Version0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.locationtech.geomesa.jobs.interop.mapred;
+
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.io.Text;
+import org.apache.hadoop.mapred.Counters.Counter;
+import org.apache.hadoop.mapred.FileOutputFormat;
+import org.apache.hadoop.mapred.JobClient;
+import org.apache.hadoop.mapred.JobConf;
+import org.apache.hadoop.mapred.MapReduceBase;
+import org.apache.hadoop.mapred.Mapper;
+import org.apache.hadoop.mapred.OutputCollector;
+import org.apache.hadoop.mapred.Reporter;
+import org.apache.hadoop.mapred.TextOutputFormat;
+import org.opengis.feature.simple.SimpleFeature;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+/**
+ * Sample job showing how to read features using GeoMesaInputFormat
+ */
+public class FeatureCountJob {
+
+    public static class Map extends MapReduceBase implements Mapper<Text, SimpleFeature, Text, Text> {
+
+        static enum CountersEnum { FEATURES }
+
+        public void map(Text key,
+                        SimpleFeature value,
+                        OutputCollector<Text, Text> output,
+                        Reporter reporter) throws IOException {
+            Counter counter = reporter.getCounter(CountersEnum.class.getName(),
+                                                  CountersEnum.FEATURES.toString());
+            counter.increment(1);
+            output.collect(new Text(value.getID()), new Text(value.getAttribute("geom").toString()));
```

Comment:
Yeah, I can change that. I'll add a comment on the expected SFT that this job runs on - it's kind of implied but not specified.